package.json

```diff
@@ -27,12 +27,11 @@
     "chalk": "^2.1.0",
     "debug": "^3.0.0",
     "generator-ibm-cloud-enablement": "0.0.109",
-    "generator-ibm-service-enablement": "0.0.89",
```

Comment:
What happened to this dependency?

Reply:
This seems odd , it may have been deleted when I `npm unlink` the service generator...I'll add it back